requirements/crosshair.in

```diff
@@ -0,0 +1,3 @@
+crosshair-tool==0.0.85
+hypothesis-crosshair==0.0.20
+-c test.in  # match test.in attrs pin
```

Comment:
I have a bad feeling about these pins.  Can we just list `hypothesis-crosshair` (no pin) and `-r test.in`, and then install from `crosshair.txt` _without_ `test.txt`?

Also happy to merge this PR and follow up later.

Reply:
I'll follow up with this tomorrow, if only to unblock CI for other PRs